package.json

```diff
@@ -23,17 +24,21 @@
     "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
     "test:watch": "npm run test -- -w ./lib",
     "lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc",
+    "prettier": "prettier --tab-width 4 --print-width 120 --single-quote --trailing-comma es5 --write ./**/**/*.js **/*.js *.js",
```

Comment:
Taking into account  .prettierrc.json I understand that `--tab-width 4 --print-width 120 --single-quote --trailing-comma es5` can be removed (as it has been done in lint-staged below).

Reply:
Fixed a573d49